packages/app-store/zoho-bigin/lib/CalendarService.ts

```diff
@@ -163,6 +163,49 @@ export default class BiginCalendarService implements Calendar {
       .catch((e) => this.log.error("Error searching contact:", JSON.stringify(e), e.response?.data));
   }
 
+  /**
+   * Generates a description from the event data which includes the Agenda,Reschedule Reason, additional notes, attendees, and user fields.
+   */
+  private getDescription(event: CalendarEvent) {
```

Comment:
**Blocking**: I think we need to revisit this function as well. I'm getting an empty string as the description

Reply:
Can you share the steps to reproduce the same? I am getting the description as expected from my side.